### PR TITLE
Update proxy example

### PR DESCRIPTION
**Pull request by danielgtaylor-isp**
I had to provide an object with routes mapped to host/port combinations to get the proxy config for the dev server working. Updated the example which I believe may have been out of date.

Awesome project, btw :+1:

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -332,7 +332,9 @@ To tell the development server to serve any `/api/*` request to your API server
 ```js
 module.exports = {
   devServer: {
-    proxy: 'http://localhost:8080/api'
+    proxy: {
+      '/api': 'http://localhost:8080'
```
Comment (egoist) [inline docs/README.md:336]: hmm I think `changeOrigin: true` is required for CORS?

**Comment by egoist** [comment]
I think we can just redirect the doc about `devServer` to https://webpack.js.org/configuration/dev-server/#devserver and then fix the behavior of `proxy: string` in Poi itself.

**Comment by danielgtaylor-isp** [comment]
@egoist ah good point about `changeOrigin`. I'm using `*` so didn't run into the issue. Do you want me to do a PR against Poi? It looks like the [path is hardcoded](https://github.com/egoist/poi/blob/b6a0d64da8338dc57b7bd5cfdd2a02ebfc1a333a/packages/poi/lib/server.js#L16-L26) for some reason so we just need to update that. For the project I'm working on, the API path is not `/api/...`.

**Comment by egoist** [comment]
yeah the `proxy: string` is convenient if your api looks like `http://localhost:8080/api/path`

**Comment by egoist** [comment]
so I think we won't need to update that part in Poi, it's just a poi-specific usage.

**Comment by egoist** [comment]
So I can actually merge this PR if you can add some notes about CORS 😄

**Comment by danielgtaylor-isp** [comment]
Closing this in favor of a proper fix to the server config. I'll try to get a PR for that shortly.

**Pull request closed without merging**